src/SonarScanner.MSBuild.Shim/ScannerEngineInput.cs

```diff
@@ -214,6 +220,9 @@ public void WriteSharedFiles(AnalysisFiles analysisFiles)
     internal void AppendKeyValue(string keyPrefix, string keySuffix, IEnumerable<string> values) =>
         AppendKeyValue($"{keyPrefix}.{keySuffix}", ToMultiValueProperty(values));
 
+    internal void AppendKeyValue(string key, IEnumerable<string> values) =>
```

Comment:
Doesn't need to be internal
```suggestion
    private void AppendKeyValue(string key, IEnumerable<string> values) =>
```

Reply:
And then I'd probably move it after `private void AppendKeyValue(string keyPrefix, string keySuffix, string value)`, before `private JProperty AppendKeyValue(string key, string value)`, so the `keyPrefix+keySuffix` ugly overloads are all together